### PR TITLE
Enable security bundle and update schema

### PR DESCRIPTION
**Pull request by paurosello**
### Checklist

- [x] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Review by T-Kukawka — APPROVED** [review]

**Comment by calvix** [comment]
what the fuck is that diff?

File: test.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apps:
+  certManager:
+    extraConfigs:
+    - kind: ConfigMap
+      name: cert-manager-cainjector
+      namespace: giantswarm
+      priority: 100
```
Comment (calvix) [inline test.yaml:7]: what is this extra file in root dir?

Comment (paurosello) [reply test.yaml:7]: shit was using it for testing :(

**Comment by paurosello** [comment]
> what the fuck is that diff?

schemagen :(

**Pull request merged**